packages/manager/apps/telecom/src/app/telecom/pack/migration/buildings/index.js

```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import uiRouter from '@uirouter/angularjs';
+import angularTranslate from 'angular-translate';
```

Comment:
It looks like `angular-translate` doesn't export the module name https://github.com/angular-translate/angular-translate/blob/master/src/translate.js#L8 so `angularTranslate` is undefined 
```suggestion
import 'angular-translate';

angular.module(moduleName, ['pascalprecht.translate'])
```